Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -109,7 +114,9 @@ def __init__(self,
                  fields_to_present: str = '',
                  csv_text: bool = False,
                  url_protocol_stripping: bool = False,
-                 url_truncate: bool = False
+                 url_truncate: bool = False,
+                 maximum_cidr_size: int = 8,
+                 no_tld: bool = False,
```

Comment:
Perhaps rename this to `no_tld_glob`.

Reply:
`no_wildcard_tld` might be a bit better (as wildcard is the common term for it)